CONTRIBUTING.md

```diff
@@ -44,12 +58,11 @@ user@user.fr => user
 
 Front assets (css/js/images) are handled by [Webpack](https://webpack.js.org/).
 
-When you're in production environment (i.e. `http://local.appbuild.com/...`), the project will use the assets found in `web/assets`, it means that you'll have to run `make assets-build` each time you edit an asset file to see the modification in your browser (after refreshing it manually). 
+When you're in production environment (i.e. `http://local.appbuild.com/...`), the project will use the assets found in `web/assets`, it means that you'll have to run `npm start` inside node container each time you edit an asset file to see the modification in your browser (after refreshing it manually). 
 
-When you're in development environment (i.e. `http://local.appbuild.com/app_dev.php/...`), you can also watch asset modifications to benefit from the hot reloading feature of webpack-dev-server, by typing the following command:
-```shell
-make assets-watch
-```
+When you're in development environment (i.e. `http://local.appbuild.com/...`), you can also watch asset modifications to benefit from the hot reloading feature of webpack-dev-server, by typing the following command:
```

Comment:
I think we don't need to maintain the documentation for both `production environment` and `development environment` since you're nginx configuration only expose the `app_dev.php` front controller anyway

this was useful previously because we had the choice between `app.php` (prod env) and `app_dev.php` (dev env)

so maybe we could remove the whole `### Frontend development` section and add a line in the `### Installation` section to say that we should run `npm start` inside the node container